### DISCUSSION TITLE
What are the use cases where restoreEl is true?

### DISCUSSION THREAD

**Discussion by startjava** [videojs/video.js#8672] [Q&A]
### Description

What are the use cases where restoreEl is true?

### Reduced test case

?

### Steps to reproduce

1.
2.
3.

### Errors

?

### What version of Video.js are you using?

?

### Video.js plugins used.

?

### What browser(s) including version(s) does this occur with?

?

### What OS(es) and version(s) does this occur with?

?

**Comment by mister-ben** [comment]
When you call `player.dispose()` the player element is removed from the DOM. With `restoreEl: true` it is replaced by a copy of the original element. It's not necessary from Video.js's point of view, but it was added as it was preferable for some 3rd party framework's notion of putting things back they were after a dispose / unmount type of action.
